src/app/api/[[...route]]/routes/task.route.ts

```diff
@@ -9,9 +13,24 @@ app.patch('/:taskId', (c) => {
   return c.json({ message: `Updating task id: ${taskId}` });
 });
 
-app.post('/:shareHouseId', (c) => {
-  const shareHouseId = c.req.param('shareHouseId');
-  return c.json({ message: `Creating task share house id: ${shareHouseId}` });
+app.post('/', zValidator('json', taskCreationSchema), async (c) => {
+  try {
+    const data = c.req.valid('json');
+    console.log('data', data);
+
+    const newTask = await prisma.task.create({
+      data: {
+        title: data.title,
+        description: data.description,
+        categoryId: data.categoryId,
```

Comment:
Could you connect the newly created task to the existing category for relations?

Reply:
I added `connect` to the category.